src/courseware/course/sequence/Sequence.jsx

```diff
@@ -144,54 +145,58 @@ const Sequence = ({
-          }}
-          previousHandler={() => {
-            logEvent('edx.ui.lms.sequence.previous_selected', 'top');
-            handlePrevious();
-          }}
-        />
-        {shouldDisplayNotificationTriggerInSequence && (
-          enableNewSidebar === 'true' ? <NewSidebarTriggers /> : <SidebarTriggers />
-        )}
-
-        <div className="unit-container flex-grow-1">
-          <SequenceContent
-            courseId={courseId}
-            gated={gated}
-            sequenceId={sequenceId}
-            unitId={unitId}
-            unitLoadedHandler={handleUnitLoaded}
-          />
-          {unitHasLoaded && (
-          <UnitNavigation
-            sequenceId={sequenceId}
-            unitId={unitId}
-            onClickPrevious={() => {
-              logEvent('edx.ui.lms.sequence.previous_selected', 'bottom');
-              handlePrevious();
-            }}
-            onClickNext={() => {
-              logEvent('edx.ui.lms.sequence.next_selected', 'bottom');
-              handleNext();
-            }}
-          />
+    <div>
+      <SequenceNavigation
+        sequenceId={sequenceId}
+        unitId={unitId}
+        className="mb-4"
+        nextHandler={() => {
+          logEvent('edx.ui.lms.sequence.next_selected', 'top');
+          handleNext();
+        }}
+        onNavigate={(destinationUnitId) => {
+          logEvent('edx.ui.lms.sequence.tab_selected', 'top', destinationUnitId);
+          handleNavigate(destinationUnitId);
+        }}
+        previousHandler={() => {
+          logEvent('edx.ui.lms.sequence.previous_selected', 'top');
+          handlePrevious();
+        }}
+      />
+      <div className="sequence-container d-inline-flex flex-row">
+        <Sidebar layout={LAYOUT_LEFT} />
+        <div className={classNames('sequence w-100 pt-3', { 'position-relative': shouldDisplayNotificationTriggerInSequence })}>
+
+          {shouldDisplayNotificationTriggerInSequence && (
+            enableNewSidebar === 'true' ? <NewSidebarTriggers /> : <SidebarTriggers />
           )}
+
+          <div className="unit-container flex-grow-1">
+            <SequenceContent
+              courseId={courseId}
+              gated={gated}
+              sequenceId={sequenceId}
+              unitId={unitId}
+              unitLoadedHandler={handleUnitLoaded}
+            />
+            {unitHasLoaded && (
+            <UnitNavigation
+              sequenceId={sequenceId}
+              unitId={unitId}
+              onClickPrevious={() => {
+                logEvent('edx.ui.lms.sequence.previous_selected', 'bottom');
+                handlePrevious();
+              }}
+              onClickNext={() => {
+                logEvent('edx.ui.lms.sequence.next_selected', 'bottom');
+                handleNext();
+              }}
+            />
+            )}
+          </div>
         </div>
+        {enableNewSidebar === 'true' ? <NewSidebar /> : <Sidebar layout={LAYOUT_RIGHT} />}
```

Comment:
[clarify]: If `enableNewSidebar` contains a boolean value, do we need to check for it to be true explicitly?

```suggestion
        {enableNewSidebar ? <NewSidebar /> : <Sidebar layout={LAYOUT_RIGHT} />}
```

Reply:
I don't know the reason of waiting string `true` value instead of boolean, but I think we should leave it without refactoring)